addons/onboarding/controllers/onboarding.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.exceptions import AccessError
+from odoo.http import request
+
+
+class OnboardingController(http.Controller):
+    @http.route(['/onboarding/<int:onboarding_id>',
+                 '/onboarding/<string:alias>'], auth='user', type='json')
+    def get_onboarding_data(self, onboarding_id=None, alias=None):
+        try:
```

Comment:
Should not need to try/except, as it should not fail and just return an empty result?